### PR TITLE
[WIP] ROC 6278 migration cleanup with dharmendrak changes

### PR DESCRIPTION
**Pull request by klawani**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-6278

### Change description ###

1) Remove usage of claim store DB & Repositories
2) Merge https://github.com/hmcts/cmc-claim-store/pull/1071 to avoid duplicate works
3) Fix some failing tests
4) Add @Ignore("to be fixed as part of task ROC-6278") to all failing integration tests which will be fixed in separate PRs to facilitate reviews 
5) Add MockedCoreCaseDataApiTest to create a set of common Wiremock Stubs  for CoreCaseDataApi endpoints

**Please note that this PR will not be merged until other PRs to fix all the failing tests have been completed.**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -144,4 +144,14 @@
         <gav regex="true">.*</gav>
         <cve>CVE-2019-3795</cve>
     </suppress>
+    <suppress>
```
Comment (sabahirfan) [inline dependency-check-suppressions.xml]: These ones are fixed and removed in master. We should not add these back.

File: charts/cmc-claim-store/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 description: Helm chart for the HMCTS CMC Claim-Store service
 name: cmc-claim-store
 home: https://github.com/hmcts/cmc-claim-store
-version: 1.1.6
+version: 1.1.7
```
Comment (sabahirfan) [inline charts/cmc-claim-store/Chart.yaml]: Is this part of this change set?

File: domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleCaseDetails.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    private static final Long CASE_ID = 1513250998636210L;
+    private static final String JURISDICTION_ID = "CMC";
+    private static final String CASE_TYPE_ID = "MoneyClaimCase";
+    private static final LocalDateTime CREATED_DATE = now();
+    private static final LocalDateTime LAST_MODIFIED = now();
+    private static final String CASE_STATE = "open";
+    private static final Integer LOCKED_BY_USER_ID = 1;
+    private static final Integer SECURITY_LEVEL = 1;
+    private static final Classification SECURITY_CLASSIFICATION = Classification.PUBLIC;
+    private static final String CALLBACK_RESPONSE_STATUS = "";
```
Comment (sabahirfan) [inline domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleCaseDetails.java]: Is this meant to be an empty string?

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/MockedCoreCaseDataApiTest.java [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+    }
+
+    @Test
+    public void shouldSuccessfullySubmitClaimForRepresentative() throws Exception {
+
+        makeSuccessfulIssueClaimRequestForRepresentative();
+    }
+
+    protected MvcResult makeSuccessfulIssueClaimRequestForRepresentative() throws Exception {
+        final ClaimData legalRepresentativeClaimData = SampleClaimData.submittedByLegalRepresentative();
```
Comment (sabahirfan) [inline src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/MockedCoreCaseDataApiTest.java]: There is no need to make local variables as `final`

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -165,9 +160,17 @@ feature_toggles:
   async_event_operations_enabled: false
   directions_questionnaire_enabled: false
 
+logging.level.uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi: DEBUG
```
Comment (sabahirfan) [inline src/main/resources/application.yml]: Do we want this change to prod?

File: src/main/resources/application.yml [hunk 6]
```diff
@@ -165,9 +160,17 @@ feature_toggles:
...
+
 feign:
   httpclient:
     enabled: true
+  client:
+    config:
+      default:
+        connectTimeout: 5000
+        readTimeout: 5000
+        loggerLevel: full
```
Comment (sabahirfan) [inline src/main/resources/application.yml]: same here

File: src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java [hunk 7]
```diff
@@ -29,8 +29,19 @@ public static UploadResponse unsuccessfulDocumentManagementUploadResponse() {
     }
 
     public static StartEventResponse successfulCoreCaseDataStoreStartResponse() {
-        String response = new ResourceReader().read("/core-case-data/start-response.success.json");
-        return jsonMapper.fromJson(response, StartEventResponse.class);
+        return jsonMapper.fromJson(startEventResponseSuccessAsFile(), StartEventResponse.class);
+    }
+
+    public static String startEventResponseSuccessAsFile() {
```
Comment (sabahirfan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java]: After introducing the builders, do we need these `JSON` resources?

Comment (klawani) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java]: removed

File: src/main/resources/application.yml (hunk 5 above)
Comment (klawani) [reply src/main/resources/application.yml]: added to trace Feign Client calls during debugging. Will not be merged to Master

File: src/main/resources/application.yml (hunk 6 above)
Comment (klawani) [reply src/main/resources/application.yml]: same as the above

**Pull request closed without merging**